terraform/ecs_ec2/daemon/variables.tf

```diff
@@ -15,7 +15,7 @@ variable "ec2_instance_type" {
 
 variable "test_dir" {
   type    = string
-  default = "./integration/test/ecs/ecs_metadata"
+  default = "./test/metric_value_benchmark"
```

Comment:
Why are we pointing to these tests instead of the ECS test?

Reply:
`metric_value_benchmark` also has the [ECS test](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/test/metric_value_benchmark/container_insights_test.go). Even though I agree you can use  `"./test/ecs/ecs_metadata"` as a test case; however, it is better to use a default test case that is supposed to be used in ECS EC2. In this case, its container insight.